core/Controller/LoginController.php

```diff
@@ -104,8 +105,7 @@ public function logout() {
 		$this->session->set('clearingExecutionContexts', '1');
 		$this->session->close();
 
-		if ($this->request->getServerProtocol() === 'https') {
-			// This feature is available only in secure contexts
+		if (!$this->request->isUserAgent([Request::USER_AGENT_CHROME, Request::USER_AGENT_ANDROID_MOBILE_CHROME])) {
```

Comment:
But something like:
```suggestion
		if ($this->request->getServerProtocol() === 'https'
			&& !$this->request->isUserAgent([Request::USER_AGENT_CHROME, Request::USER_AGENT_ANDROID_MOBILE_CHROME])) {
```

Would make sure Firefox doesn't use it on unsupported sites?

Reply:
Was added in the last commit.